sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -86,12 +86,6 @@ Licensed under the MIT License.
       </exclusions>
     </dependency>
 
-    <dependency>
```

Comment:
Any reason for removing afterburner? Was this causing any issues? It would be good to add a note to this PR so that we could refer to this later on why it was removed.

Reply:
Yes, multiple issues - https://github.com/Azure/azure-sdk-for-java/issues?q=is%3Aissue+is%3Aopen+afterburner
But we are still discussing whether to remove it, or keep it based on the performance testing.